website/assets/styles/pages/query-library.less

```diff
@@ -221,6 +238,13 @@
     box-shadow: 0 4px 16px 0 rgba(0, 0, 0, 0.1);
     margin-bottom: 90px;
     width: 100%;
+    padding: 40px;
+    p {
+      margin-block-end: 0px;
```

Comment:
@eashaw I've never seen this.  It seems to work on our supported browsers, but fyi https://caniuse.com/?search=margin-block-end